commit bd3dc51dba7993b10504cf6affd197951f85a0ff (refs/changes/94/4494/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-07-09T18:21:40-04:00 (1 year, 3 months ago)
    
    joyent/node-vasync#40 package.json repository does not point to Joyent

diff --git a/package.json b/package.json
index 60f6eee..c19b251 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
 	"main": "./lib/vasync.js",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/davepacheco/node-vasync.git"
+		"url": "git://github.com/joyent/node-vasync.git"
 	},
 	"scripts": {
 		"test": "./node_modules/.bin/tap --stdout tests/ && ./node_modules/.bin/nodeunit tests/compat.js && ./node_modules/.bin/nodeunit tests/compat_tryEach.js"
